src/metabase/formatter/datetime.clj

```diff
@@ -15,6 +15,11 @@
 
 (set! *warn-on-reflection* true)
 
+(def ^:dynamic *formatting-locale*
+  "Dynamic var to hold the current locale for datetime formatting.
+  Defaults to the site locale from system settings."
+  (Locale. (system/site-locale)))
```

Comment:
when does this diverge from 
```clojure
embedding-utils=> (doc metabase.util.i18n/user-locale)
-------------------------
metabase.util.i18n/user-locale
([])
  Locale we should *use* for the current User (e.g. `tru` messages) -- `*user-locale*` if bound, otherwise the system
  locale.
```

Reply:
Discussed [here](https://metaboat.slack.com/archives/CKZEMT1MJ/p1750874825149129)